components/cluster/command/check.go

```diff
@@ -103,9 +103,17 @@ conflict checks with other clusters`,
 				}
 			}
 
-			sshConnProps, err := cliutil.ReadIdentityFileOrPassword(opt.identityFile, opt.usePassword)
-			if err != nil {
-				return err
+			// natvie ssh has it's own logic to find the default identity_file
+			if (gOpt.NativeSSH || gOpt.SSHType == executor.SSHTypeSystem) && !utils.IsFlagSetByUser(cmd.Flags(), "identity_file") {
+				opt.identityFile = ""
+			}
+
+			var sshConnProps *cliutil.SSHConnectionProps = &cliutil.SSHConnectionProps{}
+			if gOpt.SSHType != executor.SSHTypeNone {
+				var err error
+				if sshConnProps, err = cliutil.ReadIdentityFileOrPassword(opt.identityFile, opt.usePassword); err != nil {
```

Comment:
Do we need to `cliutil.ReadIdentityFileOrPassword` even if the ssh type is `SSHTypeSystem`?

Reply:
I think we are doing this all the time.